### PR TITLE
wpc-462-add-link-input

### PR DESCRIPTION
**Pull request by HanaAyman**
This pull request is to make an input field for the links in the author edit panel appear by default

### CODE REVIEW

**Review by bishoymelek — APPROVED** [review]

File: packages/editor/src/client/panel/authorEditPanel.tsx [hunk 1]
```diff
@@ -51,7 +51,9 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
   const [jobTitle, setJobTitle] = useState<Maybe<string>>()
   const [image, setImage] = useState<Maybe<ImageRefFragment>>()
   const [bio, setBio] = useState<RichTextBlockValue>(createDefaultValue())
-  const [links, setLinks] = useState<ListValue<AuthorLink>[]>([])
+  const [links, setLinks] = useState<ListValue<AuthorLink>[]>([
+    {id: generateID(), value: {title: '', url: ''}}
```
Comment (penk85) [inline packages/editor/src/client/panel/authorEditPanel.tsx:55]: @HanaAyman why did you change the default value?

Comment (HanaAyman) [reply packages/editor/src/client/panel/authorEditPanel.tsx:55]: That's what makes it that there is always by default an empty input field for the links.

**Review by penk85 — APPROVED** [review]
